vgen/src/main/resources/io/gapi/vgen/py/main.snip

```diff
@@ -239,14 +239,14 @@
             client_config = pkg_resources.resource_string(
                 __name__, '{@jsonBaseName}_client_config.json')
             {@constructDefaults(service)}
-            google_apis_agent = '{}/{};{};gax/{};python/{}'.format(
+            goog_api_client = '{}/{};{};gax/{};python/{}'.format(
```

Comment:
Should this be space- or semicolon-separated? I've seen spaces elsewhere (e.g., Ruby)....